playkit/src/main/java/com/kaltura/playkit/plugins/KalturaLiveStatsPlugin.java

```diff
@@ -18,26 +19,42 @@
 import com.kaltura.playkit.PlaybackInfo;
 import com.kaltura.playkit.Player;
 import com.kaltura.playkit.PlayerEvent;
-import com.kaltura.playkit.Utils;
 import com.kaltura.playkit.api.ovp.services.LiveStatsService;
+import com.kaltura.playkit.plugins.configs.KalturaLiveStatsConfig;
+import com.kaltura.playkit.plugins.configs.KalturaStatsConfig;
 import com.kaltura.playkit.utils.Consts;
 
 import java.util.Date;
 import java.util.TimerTask;
 
+import static android.util.Base64.NO_WRAP;
+import static com.kaltura.playkit.plugins.KalturaLiveStatsPlugin.KLiveStatsEvent.LIVE;
+
 /**
  * Created by zivilan on 02/11/2016.
  */
 
 public class KalturaLiveStatsPlugin extends PKPlugin {
     private static final PKLog log = PKLog.get("KalturaLiveStatsPlugin");
-    private static final String TAG = "KalturaLiveStatsPlugin";
+    private static final int FIFTEEN_SEC = 15 * 1000;
```

Comment:
This const should have a meaningful name, somthing like DISTANCE_FROM_LIVE_THRESHOLD.